rpg.py

```diff
@@ -58,16 +58,56 @@ def act(self, players: List) -> List:
             players[targeted_index] = damaged_player
         return players
 
+    def view_stats(self, players):
+        for character in players:
+            if self.is_alive() and self.is_ally():
+                print(f'{character.name} LV: {character.level}')
+                print(f'HP: {character.hp}/{character.max_hp}')
+                print(f'Attack: {character.attack}')
+                print(f'Speed: {character.speed}')
+
+    def fight(self, players):
+        for character in players:
+            actions = {'fight': character.act, 'stats': character.view_stats}
+            print('What will you do?')
+            print('fight')
+            print('stats')
+            action = None
+            while action is None:
+                player_input = input('>')
```

Comment:
I tried to use the dictionary method Alex taught me with inputing enemies to attack, yet the print statement is not revealing the stats